### PR TITLE
Migrate Description #17626

### PR DESCRIPTION
**Pull request by 99rgosse**
This PR sets the parameter for migrating the description of a repository : solves #17626
Actual behavior is opts.Description is only taken in account

I had to modify services/migrations/gitea_uploader.go too as the parameter wasn't modified if not a set one.

Tested okay on latest dev

### CODE REVIEW

File: services/migrations/gitea_uploader.go [hunk 1]
```diff
@@ -109,6 +109,7 @@ func (g *GiteaLocalUploader) CreateRepo(repo *base.Repository, opts base.Migrate
 		return err
 	}
 	r.DefaultBranch = repo.DefaultBranch
+	r.Description = opts.Description
```
Comment (lunny) [inline services/migrations/gitea_uploader.go]: Or we could not change below, but only change this to `r.Description = repo.Description`?

Comment (99rgosse) [reply services/migrations/gitea_uploader.go]: oooh good one ! You're totally right...

**Comment by 99rgosse** [comment]
Sorry I may have made a mistake when force pushing to the branch, I did not see you added things. Please tell me if I have to change anything

**Comment by lunny** [comment]
> Sorry I may have made a mistake when force pushing to the branch, I did not see you added things. Please tell me if I have to change anything

Yes, you should rebase the latest main branch. Or you can create a new PR.

**Review by lunny — APPROVED** [review]

**Review by techknowlogick — APPROVED** [review]

**Comment by lunny** [comment]
make L-G-T-M work

**Comment by lunny** [comment]
Please send backport to v1.15

**Pull request merged**